Godeps/Godeps.json

```diff
@@ -8,10 +8,12 @@
 	"Deps": [
 		{
 			"ImportPath": "bitbucket.org/ww/goautoneg",
+			"Comment": "null-5",
```

Comment:
These comments are added back because the Godeps.json is now generated by `godep save`. I think the comment is innocuous.